content/en/continuous_integration/intelligent_test_runner/_index.md

```diff
@@ -10,187 +10,71 @@ further_reading:
     tag: "Blog"
     text: "Monitor all your CI pipelines with Datadog"
 ---
-{{< callout url="https://app.datadoghq.com/ci/getting-started" >}}
-Intelligent Test Runner for CI Visibility is in private beta. You can request access by completing the form on the CI Visibility Getting Started page.
-{{< /callout >}}
 
-## Overview
+<div class="alert alert-warning">Intelligent Test Runner is a beta product.</div>
 
-Intelligent Test Runner is Datadog's test impact analysis solution. It allows you to only run the impacted tests for a given commit and skip any that are irrelevant.
+Intelligent Test Runner is Datadog's test impact analysis solution. It automatically and intelligently selects and runs only the relevant tests for a given commit based on the code being changed. Significantly reduce time spent testing and overall CI costs, while maintaining test coverage.
```

Comment:
"automatically and intelligently" sounds more like marking material than docs. That's just my opinion though

Reply:
I'm sure the docs team will come up with something better but I'd rather do something like

> It uses the code changed in a given commit to automatically select and run only the relevant tests.

```suggestion
Intelligent Test Runner is Datadog's test impact analysis solution. It uses the code changed in a given commit to automatically select and run only the relevant tests. It significantly reduces time spent testing and overall CI costs, while maintaining test coverage.
```

Reply:
I see this initial text is contentious! Haha! There's already a thread about this in https://github.com/DataDog/documentation/pull/17286#discussion_r1135944042

@KickingTheTV, I think you have the last word here.